inspectit-oce-agent/src/system-test/java/rocks/inspectit/oce/metrics/GCMetricsSysTest.java

```diff
@@ -0,0 +1,46 @@
+package rocks.inspectit.oce.metrics;
+
+import io.opencensus.stats.*;
+import io.opencensus.tags.TagValue;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GCMetricsSysTest extends MetricsSysTestBase {
+
+    private static final ViewManager viewManager = Stats.getViewManager();
+
+    /**
+     * This test assumes that the JVM was started with a non-concurrent GC
+     */
+    @Test
+    public void testGCPauseCapturing() throws Exception {
+
+        //we try triggering a (non-concurrent) GC with stuff to do
+        for (int i = 0; i < 1000000; i++) {
+            new ArrayList<>();
+        }
+        System.gc();
+        Thread.sleep(1000);
```

Comment:
I think this will more or less often fail. 

According to some posts (e.g. https://stackoverflow.com/questions/1481178/how-to-force-garbage-collection-in-java) using the following two calls will increase the propability to trigger the gc:
```
System.gc ();
System.runFinalization ();
```

Or maybe doing it a few times (loop) if the test would fail..